package.json

```diff
@@ -15,11 +15,12 @@
   ],
   "scripts": {
     "postinstall": "npm ls --depth=0",
-    "start": "npm run dev --workspace app",
+    "start": "npm run dev",
     "build-release": "./bin/build-release.sh",
     "publish-release": "./bin/publish-release.sh",
     "pre-release": "./bin/pre-release.sh",
     "predev": "npm run build --workspace app",
+    "dev": "concurrently \"npm run dev --workspace app\" \"npm run dev --workspace package\" --kill-others --names \"preview,package\" --prefix-colors \"red.dim,blue.dim\"",
```

Comment:
I don't mind the `preview, package` naming here, though it's a bit of a shame to add _another_ name to argue over. Here's my 2p: `--names \"app,pkg\"`.

Reply:
The last tool like this I remember using is `npm-run-all`, which I've just seen seen was last updated in 2018, so I don't think I can offer you much better 🤣 

Looks like the maintainer is interested in the fixed-width name columns, so hopefully a feature down the line.

My comments above aren't blocking, for sure.

Reply:
This comment is marked outdated but only because it's moved to `packages/govuk-frontend-review`

I've gone with your `--names \"app,pkg\"` suggestion ✅